_posts/2015-09-04-channel.html

```diff
@@ -3699,6 +3918,959 @@
+				<td class="verb">GET</td>
+				<td><a>https://api.ustream.tv/playlists.FORMAT</a> (format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Parameters</h6>
+		<table class="table responsive">
+			<tr>
+				<th>parameter</th>
+				<th>type</th>
+				<th>importance</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td>page</td>
+				<td>integer</td>
+				<td>OPTIONAL</td>
+				<td>Requested page number (1 by default)</td>
+			</tr>
+			<tr>
+				<td>pagesize</td>
+				<td>integer</td>
+				<td>OPTIONAL</td>
+				<td>Requested page size (50 by default, max 50)</td>
+			</tr>
+			<tr>
+				<td>filter_empty_playlists</td>
+				<td>integer</td>
+				<td>OPTIONAL</td>
+				<td>If the value is 1 then empty playlists (that contain no videos) will not be returned (0 by default)</td>
+			</tr>
+		</table>
+
+		<h6>Response</h6>
+		<p>A set of a key-value structures under a "playlists" element. Example:</p>
+		<pre><code class="json">"playlists": [
+	{
+		"id": "111",
+		"title": "Title of a playlist",
+		"total_duration": "176.256",
+		"created_at": 1484127917,
+		"updated_at": 1486569501,
+		"channel_id": 11223344
+	},
+	...
+]</code></pre>
+
+		<p>Explanation of some values:</p>
+		<table class="table responsive">
+			<tr>
+				<th>key</th>
+				<th>type</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td>total_duration</td>
+				<td>float</td>
+				<td>The sum of the duration of the videos on the playlist in seconds.</td>
+			</tr>
+		</table>
+
+		<p>The paging information can be found under the "paging" element. Example:</p>
+		<pre><code class="json">"paging": {
+	"previous": ".../playlists.json?pagesize=PAGE_SIZE&page=PREVIOUS_PAGE",
+	"next": ".../playlists.json?pagesize=PAGE_SIZE&page=NEXT_PAGE",
+	"page_count": 8,
+	"item_count": 38
+}</code></pre>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>invalid_token</td>
+				<td>401 Authorization required</td>
+				<td>The provided Access Token is missing, revoked, expired or malformed</td>
+			</tr>
+		</table>
+
+		<h3>Create a playlist</h3>
+		<p>Create a new playlist for the authenticated user.</p>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">POST</td>
+				<td><a>https://api.ustream.tv/playlists.FORMAT</a> (format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Parameters</h6>
+		<table class="table responsive">
+			<tr>
+				<th>parameter</th>
+				<th>type</th>
+				<th>importance</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td><strong>title</strong></td>
+				<td>string</td>
+				<td>REQUIRED</td>
+				<td>The title of the playlist.</td>
+			</tr>
+		</table>
+
+		<p>The body of the call must contain the playlist title. The Content-Type of the request should be <strong>application/x-www-form-urlencoded</strong>.</p>
```

Comment:
Itt nem küldünk is_enabled mezőt. Ez lett: "The request body must contain the playlist title ("title" field)."